### PR TITLE
Feature/add item

### PR DESCRIPTION
**Pull request by yarivg**
image
image
image
image

### CODE REVIEW

File: src/app/items/components/edit-item-modal/edit-item-modal.component.html [hunk 1]
```diff
@@ -6,34 +6,54 @@ <h5>{{this.title}}</h5>
...
+                mdbValidate mdbInput [(ngModel)]="itemToEdit.description" formControlName="descriptionForm"></textarea>
+          <label for="descriptionForm">Description</label>
+          <mdb-error *ngIf="descriptionForm.invalid && (descriptionForm.dirty || descriptionForm.touched)">Description
+            is invalid
+          </mdb-error>
+        </div>
+        <div class="md-form md-outline">
+          <input mdbInput mdbValidate type="number" id="categoryIdForm" class="form-control"
+                 [(ngModel)]="itemToEdit.categoryId" formControlName="categoryIdForm">
+          <label for="categoryIdForm">Category ID</label>
```
Comment (Ozzorany) [inline src/app/items/components/edit-item-modal/edit-item-modal.component.html:29]: show the categories in mdb select or some dropdown and not as IDs.

- [ ] select category and not id

File: src/app/items/components/edit-item-modal/edit-item-modal.component.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+.modal-lg {
+  max-width: 70%;
+}
+
+.item-image {
+  border: 1px solid #ddd;
+  border-radius: 4px;
+  padding: 5px;
+  width: 200px;
```
Comment (Ozzorany) [inline src/app/items/components/edit-item-modal/edit-item-modal.component.scss]: try to make it responsive. use % 

- [x] check responsive

File: src/app/items/components/edit-item-modal/edit-item-modal.component.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
+.modal-lg {
+  max-width: 70%;
+}
+
+.item-image {
+  border: 1px solid #ddd;
+  border-radius: 4px;
+  padding: 5px;
```
Comment (Ozzorany) [inline src/app/items/components/edit-item-modal/edit-item-modal.component.scss]: use mdb p-2, p-3 or whatever fits

- [x] use mdb spacing

**Review by Ozzorany — CHANGES_REQUESTED** [review]

File: src/app/items/components/edit-item-modal/edit-item-modal.component.html (hunk 1 above)
Comment (yarivg) [reply src/app/items/components/edit-item-modal/edit-item-modal.component.html:29]: @TalShperling your job my friend

**Pull request merged**